prac_06/guitar_test.py

```diff
@@ -0,0 +1,29 @@
+"""
+File: guitar_test.py
+Start Time: 9:10 PM
+Finish Time: 10:45 PM
+"""
+
+from prac_06.guitar import Guitar
+
+CURRENT_YEAR = 2024
+
+
+def run_tests():
+    """Tests for Guitar class."""
+    name = "Gibson L-5 CES"
+    year = 1922
+    cost = 16035.40
+
+    guitar = Guitar(name, year, cost)
+    other = Guitar("Another Guitar", 2013, 1512.9)
+
+    print(f"{guitar.name} get_age() - Expected {102}. Got {guitar.get_age()}")
+    print(f"{other.name} get_age() - Expected {11}. Got {other.get_age()}")
+    print()
+    print(f"{guitar.name} is_vintage() - Expected {True}. Got {guitar.is_vintage()}")
+    print(f"{other.name} is_vintage() - Expected {False}. Got {other.is_vintage()}")
```

Comment:
nice,The run_tests function is well-designed and provides clear, concise tests for the Guitar class.